cli/pack/common.go

```diff
@@ -265,6 +266,13 @@ func checkTagVersionSuffix(ctx *context.Ctx) error {
 	return nil
 }
 
+func checkFilename(ctx *context.Ctx) bool {
```

Comment:
`checkFilename` is `true` if there is no `filename` and `false` otherwise. It's counterintuitive

Reply:
fixed